src/Kanbas/Courses/Quizzes/Preview/Question/List/index.tsx

```diff
@@ -1,6 +1,45 @@
-function QuizQuestionPreviewList() {
+import { IKanbasQuizQuestion } from "../../../../../store/interfaces/quizzes";
+import "./index.css";
+import { useDispatch, useSelector } from "react-redux";
+import { toggleFlagQuestion } from "../../../reducer";
+import { KanbasState } from "../../../../../store";
+import QuizQuestion from "..";
+
+function QuizQuestionPreviewList({ questionRefs }:
+    { questionRefs: React.MutableRefObject<(HTMLElement | null)[]> }) {
+
+    const dispatch = useDispatch();
+
+    const flaggedQuestions = useSelector(
```

Comment:
This doesn't need to be in redux state, as this is only applicable for the quiz preview.
Instead, use another redux state for preview components or use `useState` and pass around the state via params.

Reply:
This has not been addressed.